mlflow_project/docker_environment/DetailedConfig.md

```diff
@@ -0,0 +1,98 @@
+# [Detailed Google Kubernetes Engine (GKE) Guide](#anchor-start)
+## [Create a GKE Cluster](#anchor-create-cluster)
+__Verify Adequate GPU Quotas__
+
+Depending on your account limitations, you may have restricted numbers and types of GPUs that can be allocated within a given zone or region.
+
+- Navigate to your [GCP Console](https://console.cloud.google.com/)
+- Select "IAM & Admin" $\rightarrow$ "Quotas"  
```

Comment:
It would be good to add a screenshot of where on the console to select this (where it's located on the main page)